internal/pkg/cli/flag.go

```diff
@@ -40,7 +40,7 @@ const (
 	prodEnvFlag           = "prod"
 	deployFlag            = "deploy"
 	resourcesFlag         = "resources"
-	githubURLFlag         = "github-url"
+	repoURLFlag           = "url"
```

Comment:
Ah, that makes sense. I'm going to let `--url` have the shorthand `-u` and change the GH URL flag shorthand to `-g`; does that sound okay?

Reply:
I have that flag hidden, but added the above description anyway.

Reply:
That should be okay! or we can remove the shorthand completely for `--github-url` and it won't be a breaking change since `-u` should still work for github

Reply:
Just pushed, so will remove the shorthand for `--github-url` with the next commit.